### ISSUE TITLE
Benchmarks in documentation are out-dated

### ISSUE THREAD

**Issue by fkallen** [NVIDIA/cub#477]
- Current documentation includes benchmarks between out-dated versions of cub and thrust. (e.g DeviceReduce https://nvlabs.github.io/cub/structcub_1_1_device_reduce.html)
- Current Thrust uses cub internally for some algorithms (e.g DeviceReduce)
- It would be nice to have benchmarks on Volta and newer architectures.

**Comment by senior-zero** [comment]
This issue is blocked by the following [one](https://github.com/NVIDIA/cub/issues/631). Will update the documentation after we have tuning infrastructure in place.
